front/src/modules/trainschedule/components/ManageTrainSchedule/TimetableManageTrainSchedule.tsx

```diff
@@ -27,23 +28,25 @@ const TimetableManageTrainSchedule = ({
   infraState,
   trainIdToEdit,
   setTrainIdToEdit,
+  dtoImport,
 }: TimetableManageTrainScheduleProps) => {
   const { t } = useTranslation('operationalStudies/manageTrainSchedule');
   const [isWorking, setIsWorking] = useState(false);
 
   const leaveManageTrainSchedule = () => {
     setDisplayTrainScheduleManagement(MANAGE_TRAIN_SCHEDULE_TYPES.none);
     setTrainIdToEdit(undefined);
+    dtoImport();
```

Comment:
Since we already call `dtoImport()` in `useUpdateTrainSchedule()`, I think we don't need to call it a second time here?